packages/graph-engine/src/programmatic/input.ts

```diff
@@ -147,7 +169,7 @@ export class Input<T = any> extends Port<T> {
 		if (type.variadic) {
 			serialized.variadic = true;
 		}
-		if (type.visible == false) {
```

Comment:
This is a regression because visible can be undefined, we only want to store whether its explicitly not visible